node-js/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "crate-sample-apps",
+  "version": "1.0.0",
+  "description": "node.js support for crate-sample-apps",
+  "main": "controller.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [
+    "crate",
+    "sample"
+  ],
+  "author": "Alex, Fabian",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "express": "^4.15.3",
+    "node-crate": "^2.0.1",
```

Comment:
It looks like `node-crate` is unmaintained [^1]. What about switching to the canonical `node-postgres` [^2] today?

[^1]: https://www.npmjs.com/package/node-crate
[^2]: https://www.npmjs.com/package/pg

Reply:
Contrary to my previous assessment, it looks like `node-crate` is well alive and maintained [^1].

[^1]: https://github.com/megastef/node-crate